neo/idlib/Lib.cpp

```diff
@@ -35,7 +35,7 @@ If you have questions concerning this license or the applicable additional terms
 #include <unistd.h>
 #endif
 
-#include <SDL_endian.h>
+#include <SDL2\SDL_endian.h>
```

Comment:
this is double-wrong:
1. no backslashes in includes! (think of non-windows platforms)
2. `#include <SDL_endian.h>` really should work, if not something is set up wrong on your system

Reply:
omg! that should not have gone in